### ISSUE TITLE
Fails on multi-line value

### ISSUE THREAD

**Issue by Nuru** [mumoshu/aws-secret-operator#29]
Version 0.2.4 fails to convert an AWS secret to a Kubernetes secret if the value is multi-line:

```
$ aws secretsmanager put-secret-value --secret-id example \
--secret-string '{"example.txt": "example
multiline"}'
$  aws secretsmanager get-secret-value --secret-id example | \
grep Secret
    "SecretString": "{\"example.txt\": \"example\nmultiline\"}",
```
AWS has no problem with it. Alternately, I tried
```
$ aws secretsmanager put-secret-value --secret-id example \
--secret-string '{"example.txt": "example\nmultiline"}'
$  aws secretsmanager get-secret-value --secret-id example | \
grep Secret
    "SecretString": "{\"example.txt\": \"example\\nmultiline\"}",
```

Either way, `aws-secret-operator` fails with 
```
"error": "failed to compute secret for cr: failed to get json secret as map: invalid character \n in string literal"
```

One of the reasons to use Secrets rather than ParameterStore is that Secrets can hold big values, like TLS certificate chains. I want `aws-secret-operator` to support that.

**Comment by osterman** [comment]
@mumoshu do you have a hunch if this could be a user-error or a bug in the implementation?

**Comment by shuttie** [comment]
ASM has two formats for the secrets: key-value one and plain one. Key-value is stored as a simple JSON map, and plain one has no format at all, it's just a raw string. This operator assumes that the secret is always stored as a key-value one (so it always tries to un-marshall the json and fails).

**Comment by mumoshu** [comment]
@Nuru @osterman This has been resolved via #32 thanks to @shuttie!
